recipes/sagemaker-pytorch-container/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sagemaker-pytorch-container" %}
+{% set version = "1.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/aws/{{ name }}/archive/v{{ version }}.tar.gz"
+  sha256: f211f32ccc299714d3e6a386e7f7dbea71033d82246ad45055ea7da966f57721
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win or py==37]
```

Comment:
Same as in https://github.com/conda-forge/staged-recipes/pull/10131

Reply:
@isuruf [https://github.com/aws/sagemaker-tensorflow-container/blob/master/setup.py](url) the setup.py says to include only py27 and py36?